api/BaseAudioContext.json

```diff
@@ -510,7 +510,7 @@
               "version_added": "56"
             },
             "edge": {
-              "version_added": "≤79"
+              "version_added": "79"
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/ConstantSourceNode#browser_compatibility could also be updated to "79" then.

Reply:
I've sent https://github.com/mdn/browser-compat-data/pull/9278 to make it so.